pallets/api/src/fungibles/mod.rs

```diff
@@ -137,82 +137,83 @@ pub mod pallet {
 		) -> DispatchResult {
 			let from = ensure_signed(origin.clone())?;
 			AssetsOf::<T>::transfer_keep_alive(
 				origin,
-				asset.clone().into(),
+				token.clone().into(),
 				T::Lookup::unlookup(to.clone()),
 				value,
 			)?;
-			Self::deposit_event(Event::Transfer { asset, from: Some(from), to: Some(to), value });
+			Self::deposit_event(Event::Transfer { token, from: Some(from), to: Some(to), value });
 			Ok(())
 		}
 
-		/// Transfers `value` amount tokens on behalf of `from` to account `to`.
+		/// Transfers `value` amount tokens on behalf of `from` to account `to` with additional `data`
+		/// in unspecified format.
 		///
 		/// # Parameters
-		/// - `asset` - The asset to transfer.
-		/// - `from` - The account from which the asset balance will be withdrawn.
+		/// - `token` - The token to transfer.
+		/// - `from` - The account from which the token balance will be withdrawn.
 		/// - `to` - The recipient account.
 		/// - `value` - The number of tokens to transfer.
 		#[pallet::call_index(4)]
 		#[pallet::weight(AssetsWeightInfoOf::<T>::transfer_approved())]
 		pub fn transfer_from(
 			origin: OriginFor<T>,
-			asset: AssetIdOf<T>,
+			token: TokenIdOf<T>,
 			from: AccountIdOf<T>,
 			to: AccountIdOf<T>,
 			value: BalanceOf<T>,
 		) -> DispatchResult {
 			AssetsOf::<T>::transfer_approved(
 				origin,
-				asset.clone().into(),
+				token.clone().into(),
 				T::Lookup::unlookup(from.clone()),
 				T::Lookup::unlookup(to.clone()),
 				value,
 			)?;
-			Self::deposit_event(Event::Transfer { asset, from: Some(from), to: Some(to), value });
+			Self::deposit_event(Event::Transfer { token, from: Some(from), to: Some(to), value });
 			Ok(())
 		}
 
 		/// Approves `spender` to spend `value` amount of tokens on behalf of the caller.
 		///
 		/// # Parameters
-		/// - `asset` - The asset to approve.
+		/// - `token` - The token to approve.
 		/// - `spender` - The account that is allowed to spend the tokens.
 		/// - `value` - The number of tokens to approve.
 		#[pallet::call_index(5)]
 		#[pallet::weight(<T as Config>::WeightInfo::approve(1, 1))]
 		pub fn approve(
 			origin: OriginFor<T>,
-			asset: AssetIdOf<T>,
+			token: TokenIdOf<T>,
 			spender: AccountIdOf<T>,
 			value: BalanceOf<T>,
 		) -> DispatchResultWithPostInfo {
 			let owner = ensure_signed(origin.clone())
 				.map_err(|e| e.with_weight(Self::weight_approve(0, 0)))?;
-			let current_allowance = AssetsOf::<T>::allowance(asset.clone(), &owner, &spender);
+			let current_allowance = AssetsOf::<T>::allowance(token.clone(), &owner, &spender);
 
 			let weight = match value.cmp(&current_allowance) {
 				// If the new value is equal to the current allowance, do nothing.
 				Equal => Self::weight_approve(0, 0),
 				// If the new value is greater than the current allowance, approve the difference
-				// because `approve_transfer` works additively (see `pallet-assets`).
+				// because `approve_transfer` works additively (see `pallet-tokens`).
```

Comment:
```suggestion
				// because `approve_transfer` works additively (see `pallet-assets`).
```

Reply:
Should be pallet assets as it is the pallet's name

Reply:
Yeah, it needs changing

Reply:
Oops, thanks!